vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/Spreadsheet.java

```diff
@@ -829,6 +830,25 @@ public void setNamedRanges(List<String> namedRanges) {
                 Serializer.serialize(namedRanges));
     }
 
+    public void onPopupButtonClick(int row, int column) {
```

Comment:
Should these methods be (package) private?

Reply:
Done.